sandbox/cri/Dockerfile

```diff
@@ -0,0 +1,94 @@
+FROM rust:1.32 as base-build
+
+ENV TMP_BUILD_DIR=/tmp/build
+ENV CARGO_HOME=/usr/local/rust
+ENV RUSTUP_HOME=/usr/local/rust
+ENV RUST_VERSION=1.32.0
+ENV PATH="$CARGO_HOME/bin:$PATH"
+
+ARG FIRECRACKER_VERSION="0.15.0"
+
+# Install system dependecies
+#
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends \
+        curl \
+        make \
+        musl-tools \
+        libseccomp-dev \
+        btrfs-tools \
+        socat \
```

Comment:
Why do you need socat?

Reply:
socat is a required dependency of CRI. Something to do with portforward?

Reply:
This stage of the Dockerfile is not used to build CRI or containerd.